submit on user blur

ulferts opened the pull request:
Will also submit the work package when clicking somewhere else (blur).

https://community.openproject.com/work_packages/23113/activity

ulferts committed:
submit on user blur

TeatroIO commented:
I've prepared a stage to preview changes. <a href='http://ulferts-fix-submit-on-blur.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/' target='_blank'>Open stage</a> or <a href='http://ulferts-fix-submit-on-blur.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com//teatro.log' target='_blank'>view logs</a>.

ulferts committed:
adapt spec to save on blur

Formerly, the work package was only saved after the status field was set, now it is saved right away and we now have to add a safeguard
